package.json

```diff
@@ -1,11 +1,12 @@
 {
 	"name": "@guardian/commercial-templates-builder",
 	"version": "0.0.1",
+	"license": "Apache-2.0",
 	"scripts": {
 		"dev": "NODE_ENV=development vite dev --port 7777",
 		"build": "vite build && touch build/.nojekyll",
 		"publish": "npx gh-pages --dist build --dotfiles",
```

Comment:
Think we can also replace `npx` with `yarn` but please verify!

Reply:
Just checked and you're right, thanks Ravi!